src/Text/Pandoc/Writers/Typst.hs

```diff
@@ -223,22 +237,26 @@ blockToTypst block =
             hrows <- mapM fromRow headRows
             brows <- mapM fromRow bodyRows
             pure $ vcat (hrows ++ ["table.hline()," | not (null hrows)] ++ brows)
+      let (textstart, textend) =
+            (case formatAttrs $ pickTypstTextAttrs tabkvs of
+              [] -> ("", "")
+              tkvs -> ("#text" <> parens (literal (T.intercalate ", " tkvs)) <> "[", "]"))
```

Comment:
More idiomatic Haskell would define a function, rather than textstart and textend.

Reply:
also I don't really understand this part: why is a new `#text` being added?  Shouldn't any typst attributes attached to the table go in `table()`?